pkg/gadgets/trace/open/tracer/bpf/opensnoop.bpf.c

```diff
@@ -22,7 +23,7 @@ struct {
 	__uint(type, BPF_MAP_TYPE_HASH);
 	__uint(max_entries, 10240);
 	__type(key, u32);
-	__type(value, struct args_t);
+	__type(value, struct event);
```

Comment:
Remove `struct args_t` from opensnoop.h since it's not needed anymore after this change.